portfolio/src/main/webapp/index.html

```diff
@@ -74,6 +74,11 @@ <h3>This is a picture of a seagull.</h3>
             Louis. I found Louis at Le Mont-Saint-Michel. He posed for my picutre.
```

Comment:
Not a change in this PR but I just noticed picutre is spelled incorrectly.

Reply:
Good catch. Thanks!